hdfs_datanode/tests/common.py

```diff
@@ -42,6 +42,8 @@
     ]
 }
 
+HDFS_DATANODE_RAW_VERSION = '2.7.1'
```

Comment:
Where does this come from? Can we determine it via an env var?

Reply:
Refactored it out